e2e-tests/src/test/fee.rs

```diff
@@ -113,26 +109,17 @@ fn assert_no_scaling(
     );
 }
 
-fn prepare_transaction(connection: &SignedConnection) -> TransferTransaction {
+fn prepare_transaction<C: BalanceTransfer>(connection: &C) -> C::TransferTx {
     let bytes = [0u8; 32];
-    compose_extrinsic!(
```

Comment:
Instead `compose_extrinsic` we can use https://github.com/Cardinal-Cryptography/aleph-node/blob/main/aleph-client/src/transfer.rs#L12

Reply:
I don't see how. Transfer also sends that transaction, but this code suppose to only create it.

Reply:
I mean to refactor the code in this create so the creation of transaction object is not required here. We need such an object to send some transfers anyway, so in the end, we just need to be able to send some transfers from this created code.